src/components/member-profile/index.js

```diff
@@ -378,15 +390,25 @@ const Profile = (props) => {
               className={classNames.profilePic}
               alt={fullName}
             />
-            {isSuperUser && (
-              <div className={classNames.memberStatus}>
+            {dev && isSuperUser && isOptionKeyPressed && (
```

Comment:
too many `&&` checks here, can we have this in a variable and use that variable here?